modules/server/src/main/ts/bedrock/auto/Driver.ts

```diff
@@ -80,16 +85,19 @@ const getExtraBrowserCapabilities = (settings: DriverSettings): string[] => {
 };
 
 const getOptions = (port: number, browserName: string, settings: DriverSettings, debuggingPort: number): WebdriverIO.RemoteOptions => {
-  const options = {
-    path: settings.useSelenium ? '/wd/hub' : '/',
-    host: '127.0.0.1',
-    port,
-    logLevel: 'silent' as const,
+  const options: WebdriverIO.RemoteOptions = {
+    logLevel: 'warn' as const,
     capabilities: {
       browserName
     }
   };
 
+  if (!settings.remoteWebdriver) {
```

Comment:
Can we move this as a else statement for the if on remoteWebDriver below it seems that the settings is not really used until later. This makes the code easier to read since the branching on this is in one place.

Reply:
Changed how opts for remote are specified. Driver opts are merged before returning. Ideally, would like to have a more functional way of adding arguments to opts and capabilities but seemed like a bigger change at the time

Reply:
Yes bedrock arguments are weird and I generally hate how fragile they are